BuildLoader.py

```diff
@@ -446,6 +446,19 @@ def update_fit_table (self):
             print ('  Patching entry %d with 0x%08X:0x%08X - BIOS Module(Stage1B)' % (num_fit_entries, fit_entry.address, fit_entry.size))
             num_fit_entries     += 1
 
+            # TXT POLICY
+            if self._board.TXT_ENABLED == 1:
+                IndexPort = 0x0070
+                DataPort  = 0x0071
+                Width     = 0x01
+                Bit       = 0x04
+                Index     = 0x002a
+                addr = IndexPort + (DataPort << 16) + (Width << 32) + (Bit << 40) + (Index << 48)
+                fit_entry = FIT_ENTRY.from_buffer(rom, fit_offset + (num_fit_entries+1)*16)
+                fit_entry.set_values(addr, 0, 0, 0xa, 0)
```

Comment:
Does the CMOS address is updated after BPMGen2 signing?

Reply:
txt policy entry is present in the FIT table in the final binary after bpmgen2 signing